packages/webdriver-utils/src/util/timing.js

```diff
@@ -0,0 +1,54 @@
+import validations from './validations.js';
```

Comment:
have it in `sdk-utils`

Reply:
Tried moving basically the import statement breaks because of the nature of `validations.js` file.

Reply:
ack, lets check this next time.